plugins/backstage-plugin-bugsnag/README.md

```diff
@@ -0,0 +1,71 @@
+# Bugsnag Plugin for Backstage
+
+![a preview of the Bugsnag plugin](https://raw.githubusercontent.com/RoadieHQ/roadie-backstage-plugins/main/plugins/backstage-plugin-bugsnag/docs/backstage-bugsnag-plugin.png)
```

Comment:
Nitpick: Can we use a screenshot with the light colors instead?

Reply:
Sure, I have added it in light colors :)